src/s2/s2boolean_operation.h

```diff
@@ -26,6 +26,10 @@
 #include "s2/s2builder_layer.h"
 #include "s2/value_lexicon.h"
 
+#ifdef DIFFERENCE // Windows
```

Comment:
What is DIFFERENCE on Windows? Maybe we should rename this one instead. @ericveach

Reply:
I don't currently have access to the headers to link/paste here, but I believe it was for something intended to be used primarily inside the header itself, some kind of API versioning deal. It shouldn't be a problem to undef it here, but you could change your own enum instead if you wish.